packages/jest-runner/src/index.ts

```diff
@@ -98,12 +101,14 @@ class TestRunner {
     onResult: OnTestSuccess,
     onFailure: OnTestFailure,
   ) {
-    const worker: WorkerInterface = new Worker(TEST_WORKER_PATH, {
+    const worker = new Worker(TEST_WORKER_PATH, {
       exposedMethods: ['worker'],
-      forkOptions: {stdio: 'pipe'},
+      // TODO: clarify this in PR
+      // doing this to satify the type
+      forkOptions: {stdio: ['pipe']},
```

Comment:
The type expects an array. The old code uses a string

Reply:
that's wrong in the types, the correct ones are documented here: https://nodejs.org/api/child_process.html#child_process_options_stdio

Reply:
Sure thing, I'll revert this and add a `ts-ignore` for now

Reply:
fixed the typings in `jest-worker` 🙂